experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -252,11 +270,71 @@ def post(self, *args, **kwargs):
         return response
 
 
+class NimbusExperimentsCloneView(NimbusExperimentViewMixin, RequestFormMixin, CreateView):
```

Comment:
Oh okay, so what you could do here is instead of using `CreateView` you could use `UpdateView` which would then include the parent slug in the url and you could access the parent via `self.object`